core/internal/cltest/mocks.go

```diff
@@ -334,21 +334,26 @@ func fwdHeaders(actual, mock interface{}) {
 
 // MockSubscription a mock subscription
 type MockSubscription struct {
-	name    string
-	channel interface{}
-	Errors  chan error
+	name         string
+	channel      interface{}
+	unsubscribed bool
+	Errors       chan error
 }
 
 // EmptyMockSubscription return empty MockSubscription
-func EmptyMockSubscription() MockSubscription {
-	return MockSubscription{Errors: make(chan error, 1), channel: make(chan struct{})}
+func EmptyMockSubscription() *MockSubscription {
```

Comment:
Forgive my ignorance, why is it a pointer now?

Reply:
Good question, I’ve updated the commit message accordingly.

It was to ensure changes to member variables are persisted after method calls.  The methods were defined on the concrete value, and it was causing issues.